data.sql

```diff
@@ -45,4 +45,35 @@ WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 
 UPDATE animals 
 SET owner_id=5 
-WHERE name IN ('Angemon','Boarmon');
\ No newline at end of file
+WHERE name IN ('Angemon','Boarmon');
+
+INSERT INTO vets (id, name, age, date_of_graduate) VALUES (1, 'William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (id, name, age, date_of_graduate) VALUES (2, 'Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets (id, name, age, date_of_graduate) VALUES (3, 'Stephanie Mendez ', 64, '1981-05-04');
+INSERT INTO vets (id, name, age, date_of_graduate) VALUES (4, 'Jack Harkness', 38, '2008-05-08');
+
+INSERT INTO specializations (vets_id, species_id) VALUES (1, 1);
+INSERT INTO specializations (vets_id, species_id) VALUES (3, 2);
+INSERT INTO specializations (vets_id, species_id) VALUES (3, 1);
+INSERT INTO specializations (vets_id, species_id) VALUES (4, 2);
+
+INSERT INTO visits (animals_id, vets_id, date) VALUES (1, 1, '2000-05-24');
```

Comment:
[Optional]
- [ ] You should make a query with the name instead of setting the id. 
On actual-world projects, there is no way to know all ids for all records.

For Example : 
```sql
INSERT INTO specializations (species_id, vet_id) VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));


INSERT INTO visits (animal_id, vet_id, vist_date) VALUES ((SELECT id FROM animal WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24');
```

Reply:
Thank you very much.